packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -38,13 +38,16 @@ const farmExpenseTypeController = {
           } else {
             // if its deleted, them make it active
             record.deleted = false;
+            record.custom_description = data.custom_description || undefined;
```

Comment:
**This is an optional suggestion, it doesn't change functionality!**

Now that I understand what the goal is with in the database column (`null` if it's not a string value, and no need to distinguish between API and frontend), I think the more common pre-existing pattern would be to clean it up between the `onSubmit` and the saga in the frontend, e.g.:

```js
  const onSubmit = (data) => {
    const payload = {
      ...data,
      custom_description: data.custom_description || undefined, // or null
    };
    dispatch(addCustomExpenseType(payload));
  };
  ```
 
If it stays in the controller you can move the `|| undefined` line to the top so it doesn't have to be duplicated in both logical branches.